src/theme.jsx

```diff
@@ -36,16 +36,17 @@ const theme = createMuiTheme({
     },
   },
   typography: {
+    useNextVariants: true,
     ...Roboto400,
-    display4: Roboto300,
-    display3: Roboto400,
-    display2: Roboto400,
-    display1: Roboto400,
-    headline: Roboto400,
-    title: Roboto500,
-    subheading: Roboto400,
-    body2: Roboto500,
-    body1: Roboto400,
+    h1: Roboto300,
+    h2: Roboto400,
+    h3: Roboto400,
+    h4: Roboto400,
+    h5: Roboto400,
+    h6: Roboto500,
+    subtitle1: Roboto400,
+    body1: Roboto500,
+    body2: Roboto400,
```

Comment:
Now that we are using the next variants, we'll need to update the codebase. The new mapping is:

display4 => h1
display3 => h2
display2 => h3
display1 => h4
headline => h5
title => h6
subheading => subtitle1
body2 => body1
body1 (default) => body2 (default)

This means something like

```jsx
<Typography variant="display4" ... />
``` 

should become

```jsx
<Typography variant="h1" ... />
```